CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add vignette for using the plot_bands() function (@nkorinek, #315)
 * Add example vignette for stacking and cropping raster bands with EarthPy (@nkorinek, #267)
 * Add example vignette for plotting band combinations with EarthPy (@jlpalomino, #306)
+* Add example vignette for using the hillshade function (@nkorinek, #329)
```

Comment:
@nkorinek given this is the most recent addition, please add this to the top of the unreleased list rather than the bottom